src/Compilers/Core/Desktop/CommandLine/CommandLineAnalyzerDependency.cs

```diff
@@ -0,0 +1,38 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+using Roslyn.Utilities;
+
+namespace Microsoft.CodeAnalysis
+{
+    /// <summary>
+    /// Describes a command-line analyzer assembly dependency specification.
+    /// </summary>
+    public struct CommandLineAnalyzerDependency : IEquatable<CommandLineAnalyzerDependency>
+    {
+        public CommandLineAnalyzerDependency(string path)
+        {
+            FilePath = path;
+        }
+
+        /// <summary>
+        /// Assembly file path.
+        /// </summary>
+        public string FilePath { get; }
+
+        public override bool Equals(object obj)
+        {
+            return obj is CommandLineAnalyzerDependency && base.Equals((CommandLineAnalyzerDependency)obj);
```

Comment:
@tmat This follows the pattern in `CommandLineAnalyzerReference`. Do you remember why that type calls `base.Equals(...)` instead of calling the `Equals(CommandLineAnalyzerReference other)` method directly?